manifest.yml

```diff
@@ -0,0 +1,6 @@
+---
+applications:
+  - name: carbon-tutorial-DanielLee12
+    memory: 64M
+    buildpack: https://github.com/cloudfoundry/staticfile-buildpack.git
+    REACT_APP_GITHUB_PERSONAL_ACCESS_TOKEN: 989d0967550b7e949dabb607c2188fa97255cb7a
```

Comment:
It's good practice not to add access tokens into source control.  The token isn't needed here, and might cause your token to get revoked.